### DISCUSSION TITLE
Pocket base in production ?

### DISCUSSION THREAD

**Discussion by gokulgseven** [pocketbase/pocketbase#4312] [Q&A]
I made a small file storage app using pocketbase , users can upload files and then access them later . 
will be used by some users 

what are the things that i have to take care of while using pocketbase in production / only from the security prospective? 

things that i have narrowed down till now that i should be concerned about ? 
1) XSS CRSF  https://github.com/pocketbase/js-sdk?tab=readme-ov-file#security 
2) sql injection : prevented by binding parameters in any custom Api : https://pocketbase.io/docs/js-database/
3) brute force : prevented by rate limiting on nginx reverse proxy 

is there any other thing that I should be aware of 
(my background is express backend development, these are all the things that happened automatically using Orm and libraries in express and the developers didn't have to do much)

Are there other such security concerns that i am missing ? 
(anything other than the api rules and permissions, and the business login)

**Comment by ollema** [comment]
https://pocketbase.io/docs/going-to-production/

**Reply by gokulgseven** [reply]
thank you, but i am looking for things that impact security only

**Comment by LickABrick** [comment]
Quite a obvious one, but make sure you setup your API permissions correctly in PB.
